apps/air-discount-scheme/web/lib/environment.ts

```diff
@@ -1,21 +1,30 @@
+import { getStaticEnv } from '@island.is/shared/utils'
 const isProd = process.env.NODE_ENV === 'production'
 
 const devConfig = {
   production: false,
-  NEXTAUTH_URL: process.env.NEXTAUTH_URL ?? 'http://localhost:4200',
-  identityServerDomain:
-    process.env.IDENTITY_SERVER_DOMAIN ?? 'identity-server.dev01.devland.is',
-  identityServerLogoutURL: process.env.NEXTAUTH_URL ?? 'http://localhost:4200',
+  NEXTAUTH_URL: 'http://localhost:4200',
+  identityServerDomain: 'identity-server.dev01.devland.is',
+  identityServerLogoutURL: 'http://localhost:4200',
   identityServerSecret: process.env.IDENTITY_SERVER_SECRET,
   idsTokenCookieName: 'next-auth.session-token',
   csrfCookieName: 'next-auth.csrf-token',
 }
 
+if (isProd) {
+  if (process.env.NEXT_PUBLIC_NEXTAUTH_URL === undefined) {
+    throw new Error('Missing Nextauth_Url in web environment.')
+  }
+  if (process.env.NEXT_PUBLIC_IDENTITY_SERVER_DOMAIN === undefined) {
+    throw new Error('Missing IDS domain in web environment.')
+  }
+}
```

Comment:
I think it is best to remove this code.

Reply:
First I need to get the infra env values to work without breaking build.